CONTINUITY.md

```
@@ -1,60 +1,62 @@
 ## Estado actual
 
-### Completado
-- PR #166 mergeado — pipeline CFDI Recibidos completo en main
-- issue #152 cerrado
-- sync-check.md actualizado (frappe-infrastructure `c055158`) con 3 reglas nuevas
-
-### PR #167 — en revisión
-- 13 docs marcados OBSOLETO + archivados
-- 5 docs archivados sin nota
-- 18 archivos movidos a docs/development/archive/
+### Completado en esta sesión
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)